subsys/bluetooth/host/att.c

```diff
@@ -61,6 +61,9 @@ NET_BUF_POOL_DEFINE(prep_pool, CONFIG_BT_ATT_PREPARE_COUNT, BT_ATT_MTU,
 		    sizeof(struct bt_attr_data), NULL);
 #endif /* CONFIG_BT_ATT_PREPARE_COUNT */
 
+K_MEM_POOL_DEFINE(req_pool, sizeof(struct bt_att_req),
+		  sizeof(struct bt_att_req), CONFIG_BT_ATT_TX_MAX, 16);
```

Comment:
Wouldn't `k_mem_slab` be the more appropriate data structure to use since you're allocating fixed same-sized blocks of data? Using `k_mem_pool` has the added cost of supporting arbitrary-sized allocations which you're not taking advantage of.

Reply:
Will change to k_mem_slab.

Reply:
Done.